vimeo-networking/src/main/java/com/vimeo/networking/model/VideoFile.java

```diff
@@ -88,6 +88,9 @@ public String toString() {
     private VideoQuality quality;
     private MimeType type;
     public VideoLog log;
+    /** The md5 provides us with a way to uniquely identify video files at {@link #link} */
+    @SerializedName("md5")
+    public String md5;
```

Comment:
We should probably be using `@SerializedName` on every field going forward because of how GSON's versioning works. If we persist any object without it, and we rename the field then it will be broken forever. But as long as the annotation never changes, we can rename until the cows come home without any worry a versioning incompatibility.
So it's just a good way to future proof ourselves and we can only really benefit from it.
It's unlikely that our api will change the field names - but it's also totally possible.
Kind of as you said, it couldn't hurt.